### PR TITLE
fix(config): the sample every new user copies was 3.5x over the free tier

### PR DESCRIPTION
**Pull request by tarigelamin1997**
## What was wrong

`config/search_config.sample.yml` is the template [getting-started §6](docs/getting-started.md) tells **every new user** to copy. It was sized far over JSearch's free tier:

| | titles × countries × pages | per sweep | per month |
|---|---|---|---|
| **Old sample** | 3 × 6 × 5 (capped by budget 70) | 70 | **~700 — 3.5× over** |
| Live config / **new sample** | 3 × 5 × 1 | 15 | **≤ 165** (worst month) |

Its own comment read `# fits free 200/mo`, and getting-started repeated *"fits JSearch's free tier."* Anyone following the docs would have run out within about a week and hit [ERR-017](docs/ledgers/errors.md) — the outage this project spent two months in.

## The fix

- **The sample now matches the live config**, which is already proven to fit.
- **The `budget:` block explains the arithmetic in plain terms** — `titles × countries × pages × ~11`, pages as the hidden multiplier, and what to do if you want more: cut something else, or move to a paid RapidAPI plan and raise the numbers *together*.
- **Getting-started points at that one block** instead of restating the numbers (one canonical home per fact).

## The claim is a gate now, not a comment

New tests in `tests/test_search_spec.py` fail if the sample stops fitting the free tier in its **worst** month (11 sweeps, not the average of 10), or if its budget falls below one full sweep. Each negative runs through the same predicate as its positive.

**Mutation-checked** (sample restored after each):

| Mutant | Result |
|---|---|
| Old shape: 5 pages, budget 70 | 2 tests fail |
| A 4th title → 20/sweep, 220/month | 2 tests fail |
| **Oman back at 1 page → 18/sweep** | **1 test fails** |

That third one is why the budget test exists on its own: **198/month fits the quota**, but budget 16 is below the 18-request sweep, so every sweep would stop early as `budget_exhausted`. The quota test alone would have passed it.

## Close-out judgement, in writing

> *Did this change a surface, a resource, or a behaviour a first-time reader must know about?*

**It changes what a first-time user is told and the config they copy** — so `getting-started.md` is updated. `README.md` and `CLAUDE.md` already describe the live `3 × 5 × 1` config correctly; only their guarded test counts moved. No new surface, resource, migration or dependency. **No live config change** — the deployed `search_config.yml` is untouched, so the 2026-09-22 validation is unaffected.

## Gate

ruff clean · both doc gates pass · **619 unit tests pass**. Integration not run locally (no DB code touched; Docker off) — CI runs it.

**Review status:** a fresh Examiner is auditing this change now (every figure recomputed, mutation-tested). I'll post its verdict here before this is merge-ready.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

- **Configuration**
  - Updated sample search settings to one page per query and a lower request budget.
  - Removed Oman from the sample country list.
  - Clarified request costs, free-tier limits, search cadence, and quota exhaustion.

- **Documentation**
  - Clarified that default settings are designed to remain within the monthly free tier.
  - Updated test totals to 622 unit, 68 integration, and 690 total tests.

- **Tests**
  - Added coverage for monthly request boundaries and configured search cadence.

### CODE REVIEW

**Comment by tarigelamin1997** [comment]
## Examiner verdict: 0 blockers · 1 should-fix · 3 minors — all fixed

A fresh Examiner recomputed every figure in this PR (all arithmetically true; the ERR-017 evidence is paraphrased faithfully), confirmed a sweep can never bill **more** than `titles × countries × pages` — so the gate errs conservative — and mutation-tested the new tests.

| # | Finding | Fix | Mutation check |
|---|---|---|---|
| **S1** | "Worst month, not average" was unpinned: swapping `ceil(31/N)` for `30//N` left all four new tests green | Boundary case — 20/sweep is 200 in an average month (fits), 220 in the worst (fails) | average → **boundary test fails** |
| **M1** | The 165 figure and "Enforced" held only for the build constant; `lambda.tf` set to `"2"` needs ~240 with everything green | A test pins `lambda.tf`'s cadence to the constant the arithmetic uses; scope stated in the sample and getting-started (manual runs on a fetch day spend extra) | cadence → 2: **fails** · variable renamed: **fails loudly** |
| **M2** | "~700", "3.5×", "about a week" were budget-cap ceilings | "up to ~700–770", "as much as 3.85×", "first week or two" | wording |
| **M3** | The build plan still described the old sample | Dated supersession note; history kept | docs |

**Re-verification:** each code fix was mutation-checked, files restored byte-for-byte. The fixes are small (three tests plus wording), so I did not spawn a second Examiner — say if you want one.

**Gate:** ruff clean · both doc gates pass · **622 unit tests pass**.

**Comment by tarigelamin1997** [comment]
@coderabbitai review

File: docs/04-v0-build-plan.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ EventBridge (daily) → **one Lambda**: fetch from **one source** → land raw i
...
+
+> **Superseded 2026-09-14 — the sample no longer matches this paragraph.** The matrix above was the Step-0 *probe*, sized for two or three one-off sweeps. [`config/search_config.sample.yml`](../config/search_config.sample.yml) is now the free-tier-safe *daily* default: **3 titles × 5 countries × 1 page = 15 per sweep**, at most 165 a month on the scheduled runs at the default cadence. Oman was dropped on its measured hit rate ([ERR-017](ledgers/errors.md)). This paragraph is kept as written because it records what Step 0 actually ran.
```
[the remark this answers is not in the document]
Comment (tarigelamin1997) [reply docs/04-v0-build-plan.md]: Fixed in 101a247: the note now calls the sample the free-tier-safe default for the daily Lambda, and says ingest() runs the JSearch sweep every third day. The 165/month figure and the Step-0 history are unchanged.
